src/TASOPT.jl

```diff
@@ -33,10 +34,8 @@ export convertMass, convertForce, convertDist,
 
 include("./misc/index.inc")
 include("./misc/aircraft.jl")
-export aircraft
+export aircraft, fuselage_tank
```

Comment:
Nicer to put `fuselage_tank` in it's own file.

Reply:
^